docs/htdocs/info/docs/tools/vep/script/vep_tutorial.html

```diff
@@ -271,6 +271,16 @@ <h2 id="run">Run VEP</h2>
     
     <p> We can see frequency data for the AFR, AMR, EAS, EUR and SAS continental population groupings; these represent the frequency of the alternate (ALT) allele from our input (G in the case of rs7289170). Note that the Existing_variation column is populated by the identifier of the variant found in the VEP cache (and that it corresponds to the identifier from our input in Uploaded_variation). To retrieve only this information and not the frequency data, we could have used <a href="vep_options.html#opt_check_existing">--check_existing</a> (--af_1kg silently switches on --check_existing). </p>
    
+    <div>
```

Comment:
I'd go for something less prominent here, since most people have not had problems with it. We could tweak the paragraph above with something like:
VEP can **be configured to** annotate your input with identifiers and frequency data from variants co-located with your input data. 
istead of-
VEP can annotate your input with identifiers and frequency data from variants co-located with your input data.